### PR TITLE
Newsletter effects

### PR DESCRIPTION
**Pull request by Nolan-Arnold**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Fixes: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

## Other information

### CODE REVIEW

File: libs/newsletter/src/facades/newsletter.facade.ts [hunk 1]
```diff
@@ -7,7 +7,8 @@ import { DaffNewsletterModule } from '../newsletter.module';
 import { isObject } from 'util';
 
 import { State, selectDaffNewsletterSuccess, selectDaffNewsletterError, selectDaffNewsletterLoading } from '../selectors/newsletter.selector';
-import { DaffStoreFacade } from '@daffodil/core';
+import { DaffStoreFacade } from 'libs/core/src';
```
Comment (damienwebdev) [inline libs/newsletter/src/facades/newsletter.facade.ts]: @Nolan-Arnold needs fix.

File: libs/newsletter/testing/src/drivers/testing/newsletter.service.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { DaffNewsletterServiceInterface } from "libs/newsletter/src/driver/interfaces/newsletter-service.interface";
```
Comment (damienwebdev) [inline libs/newsletter/testing/src/drivers/testing/newsletter.service.ts]: Needs path fix.

File: libs/newsletter/src/effects/newsletter.effects.spec.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import { DaffNewsletterEffects } from "./newsletter.effects";
+import { DaffNewsletterSubmission } from "../models/newsletter.model";
+import { Observable, of } from "rxjs";
+import { DaffNewsletterServiceInterface } from "../driver/interfaces/newsletter-service.interface";
+import { TestBed } from "@angular/core/testing";
+import { DaffNewsletterDriver } from "../driver/injection-tokens/newsletter-driver.token";
+import { provideMockActions } from "@ngrx/effects/testing";
+import { DaffNewsletterSubscribe, DaffNewsletterSuccessSubscribe, DaffNewsletterFailedSubscribe } from "../actions/newsletter.actions";
+import { hot, cold } from "jasmine-marbles";
+import { DaffTestingNewsletterService } from "libs/newsletter/testing/src/drivers/testing/newsletter.service";
```
Comment (damienwebdev) [inline libs/newsletter/src/effects/newsletter.effects.spec.ts]: Needs path fix.

File: libs/newsletter/src/effects/newsletter.effects.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+import { DaffNewsletterDriver } from '../driver/injection-tokens/newsletter-driver.token';
+
+export class DaffNewsletterEffects<T extends DaffNewsletterSubmission, V>{
+
+  constructor(
+    private actions$: Actions,
+    @Inject(DaffNewsletterDriver) private driver: DaffNewsletterServiceInterface<T, V>) { }
+
+  @Effect()
+  trySubmission$ = createEffect(() => this.actions$.pipe(
```
Comment (damienwebdev) [inline libs/newsletter/src/effects/newsletter.effects.ts]: Since these functions do the same thing, would it make sense to use two arguments to `ofType`?

Comment (Nolan-Arnold) [reply libs/newsletter/src/effects/newsletter.effects.ts]: I would agree with that, however I wasn't sure if `ofType` could accept two arguments.

**Pull request merged**